services/core-web/src/styles/components/Forms.scss

```diff
@@ -27,4 +28,55 @@
 
 .has-error .ant-checkbox-group:not([disabled]):hover {
   border-color: #ff4d4f;
+}
+
+// FORMS
```

Comment:
was having a hard time deciding when to put things in antd-overrides and when to put them in Forms, so just moved over from overrides to the more specific file.

Reply:
In the future, might want to think about implementing some common styles, especially for basic layout type stuff.